north_tests/src/process_assert.rs

```diff
@@ -52,35 +52,35 @@ impl ProcessAssert {
         let mut cgroup_path = self
             .get_cgroup_path("cpu")
             .await
-            .context("Could not find the cgroup path")?;
+            .wrap_err("Could not find the cgroup path")?;
         cgroup_path.push("cpu.shares");
 
         fs::read_to_string(cgroup_path.as_path())
             .await
             .map(|s| s.trim().to_owned())?
             .parse()
-            .context("Could not parse CPU shares")
+            .wrap_err("Could not parse CPU shares")
     }
 
     /// Return the limit in bytes set in the process's memory cgroup
     pub async fn get_limit_in_bytes(&self) -> Result<u64> {
         let cgroup_path = self
             .get_cgroup_path("memory")
             .await
-            .context("Could not find the cgroup path")?
+            .wrap_err("Could not find the cgroup path")?
```

Comment:
Nitpick: the common error term in northstar is `Failed to ...` instead of `Could not...`